Recrutify/Recrutify.Services/Services/FormEmailService.cs

```diff
@@ -1,36 +1,83 @@
 ﻿using System.Collections.Generic;
 using System.IO;
 using Mustache;
+using Recrutify.Services.DTOs;
 using Recrutify.Services.EmailModels;
 using Recrutify.Services.Services.Abstract;
 
 namespace Recrutify.Services.Services
 {
     public class FormEmailService : IFormEmailService
     {
-        public FormEmailService()
+        public IEnumerable<EmailRequest> FormAcceptanceEmail(List<CandidateDTO> candidates)
```

Comment:
these three methods are almost the same, make 4th private method FormStatusChangedEmail, and pass required parameters to it